recipes/openblas/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+:: Set $HOME to the current dir so msys runs here
+set HOME=%cd%
+
+:: Configure, build, test, and install using `make`.
+bash -lc "make FC=gfortran DYNAMIC_ARCH=1 BINARY=$ARCH NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 PREFIX=`echo $LIBRARY_PREFIX | tr '\\' '/'`"
```

Comment:
Windows is a weird place for sure. Do we know if openblas is going to be supporting msvc better in the future?

BTW was there a decision made regarding msys2 and if we should be moving away from msvc anyway? Sorry I seem to have lost track of the discussion about this.